#goal definition
---
#result definition
int32 images_acquired
---
#feedback
int32 images_acquired